MVP/defaults-build.yaml

```diff
@@ -3,23 +3,9 @@
     node: 'jslave-platform'
     parameters:
         - string:
-            name: id
-        - string:
-            name: task_id
-        - string:
-            name: CI_TYPE
+            name: MESSAGE_HEADERS
         - string:
             name: CI_MESSAGE
-        - string:
-            name: tag
-        - string:
-            name: name
```

Comment:
These are helpful for checking or tweaking parameters from a build (e.g. resubmit a test)
I don't see any benefit by removing them.

Reply:
Added 'tag', 'name', 'version' and 'release' back as they can be found in the message header of a kernel candidate UMB message.